editor/ok_interface.py

```diff
@@ -17,13 +17,6 @@
 
 FAILURE_SETUP_FOOTER = "; Raw ok output over"
 
-##############
```

Comment:
Why these changes?

Reply:
`py-backwards` messes up the order of imports for some reason, so they go before the path is updated. So I moved the imports inside the function call, and now it works correctly.

Reply:
Huh interesting.